pages/docs/concepts/asyncapi-document/securing-operations.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Securing Operations'
+weight: 120
+---
+
+In an AsyncAPI document, the concept of server security implies that the security measures defined at the server level apply to all operations within all channels by default. 
+
+## Features of Security in AsyncAPI Documents
+
+- The security requirements specified at the server level are enforced consistently across the entire asyncapi document.
+
+There may be situations where certain operations within specific channels require different security measures than the default server-level settings. To accommodate for such scenarios, AsyncAPI document allows you to use the `security` property at the `operation` level.
+
+By using the `security` property at the `operation` level, you can override the default server-level security and define unique security requirements for individual operations. Thereby also giving you the flexibility to tailor the security measures to the specific needs of each operation, even if they differ from the broader server-level settings.
+
+## Specifying Security Requirement 
+
+To specify different security requirements for a specific operation, you can include the `security` property within the operation's definition. 
+The security property is an array where you can define one or more security requirement objects.
+
+For example, let's say you have an AsyncAPI document with a channel called users and two operations within that channel: createUser and getUser. 
+The server-level security is set to use API key authentication for all operations within all channels. 
```

Comment:
you cannot enforce as it is not about overriding, 

it is like providing alternative for what is already in the server. Of course you can do a server without security, then have different operations and channels on the same server, each operation with different security - still, it is not overriding/enforcing if you know what I mean

Reply:
I am unable to understand how to improve the content as per your suggestions.
could you please attach some examples or maybe give me a bit more info about the same?

Reply:
> you cannot enforce as it is not about overriding,
> 
> it is like providing alternative for what is already in the server. Of course you can do a server without security, then have different operations and channels on the same server, each operation with different security - still, it is not overriding/enforcing if you know what I mean

I have removed the line that stated about enforcing/overriding, though not sure if I still was able to resolved the suggestion.

I have implemented the other changes suggested, do please let me know if I need to add/remove any thing else.